transmart-core-db/src/main/groovy/org/transmartproject/export/DataTypes.groovy

```diff
@@ -0,0 +1,13 @@
+package org.transmartproject.export
+
+import org.transmartproject.core.ontology.OntologyTerm
+
+class Datatypes {
+
+    Map<Integer, List<OntologyTerm>> OntologyTermsMap = [:]
```

Comment:
Start field names with lowercase please.
Please add comment describing what is being mapped to what. In this case cohort ids are being mapping to concepts?